src/qililab/qprogram/qblox_compiler.py

```diff
@@ -400,7 +402,22 @@ def __handle_static_sync(self, buses: set[str]):
     def __handle_dynamic_sync(self, buses: set[str]):
         raise NotImplementedError("Dynamic syncing is not implemented yet.")
 
+    def _handle_measure(self, element: Measure):
+        """Wrapper for qblox play and acquire methods to be called in a single operation for consistency with QuantumMachines
+        measure operation
+
+        Args:
+            element (Measure): measure operation
+        """
+        if not isinstance(element.weights, IQPair):
+            raise NotImplementedError("Qblox measure operation only supports weight format as IQPairs")
+        play = Play(bus=element.bus, waveform=element.waveform)
+        acquire = Acquire(bus=element.bus, weights=element.weights)
+        self._handle_play(play)
+        self._handle_acquire(acquire)
```

Comment:
The deletion of `Acquire` and `_handle_acquire` will be done in a later PR, right?

Reply:
Also in the final version, I would prefer all the logic to be in the `_handle_measure`  directly. (without creating new objects or calling other handlers manually) If there is duplicate code (e.g. making the Q1ASM play instruction) we should create a separate method that does that.

This is both for consistency with the rest of the code and also that probably you will need to do different things in creating the Q1ASM for Play and for Measure. E.g. Play won't have a `time_of_flight` while Measure will have.

Reply:
> The  deletion of Acquire and _handle_acquire will be done in a later PR, right?

this is what I discussed at:
https://linear.app/qilimanjaro/issue/QHC-262/delete-acquire-and-implement-measure-in-qbloxcompiler#comment-fc64d49c

If we add this and ToF at the same time then I don't mind deleting `handle_aqcuire`